src/main/java/kitchenpos/domain/OrderLineItem.java

```diff
@@ -6,6 +6,16 @@ public class OrderLineItem {
     private Long menuId;
     private long quantity;
 
+    public OrderLineItem() {
```

Comment:
```suggestion
    public OrderLineItem() {}
```
이 부분은 중요한 것은 아니지만 [컨벤션](https://naver.github.io/hackday-conventions-java/)에 따르면 이렇게 써줄 수도 있을 것 같아요...!